packages/discord.js/src/managers/GuildChannelManager.js

```diff
@@ -51,10 +51,7 @@ class GuildChannelManager extends CachedManager {
    * @readonly
    */
   get channelCountWithoutThreads() {
-    return this.cache.reduce((acc, channel) => {
-      if (ThreadChannelTypes.includes(channel.type)) return acc;
-      return ++acc;
-    }, 0);
+    return this.cache.filter(channel => !ThreadChannelTypes.includes(channel.type)).size;
```

Comment:
This increases memory usage unnecessarily, as filter creates a new collection we immediately dismiss again. The prior solution didn’t. For the scope of this PR a simple change of `++acc` to `acc+1` would be better.